include/wlr/render/egl.h

```diff
@@ -24,6 +24,7 @@
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
 #include <pixman.h>
+#include <gbm.h>
```

Comment:
We don't actually need to include this header, I think. It should work fine without.

Reply:
I moved it to the `egl.c` file where it's needed for the `gbm_destroy_device` call

Reply:
Cool. It seems this line was still added in this commit (and removed in the next one), fixed it locally.